lib/iris/fileformats/pp.py

```diff
@@ -1591,20 +1598,31 @@ def _interpret_fields(fields):
                 (field.lbuser[3] % 1000) == 30:
             land_mask = field
 
+        apply_landmask = None
```

Comment:
@pp-mo Minor point - The PP loading logic is pretty heavy going (always has been). So to help the reader I think it would be make a stronger association that `apply_landmask = None` is the `else` of `if (field.raw_lbpack // 10 % 10) == 2:` ... would you mind doing that?

Reply:
Also change `apply_landmask` to `land_mask_field`, after all that's what it actually is. The use of `apply_landmask` subtly hints (to me) that it's a boolean, and its not.